### PR TITLE
Updated release procedures.

### PR DESCRIPTION
**Pull request by tonya1**
This change is [Reviewable](https://reviewable.io/reviews/gridappsd/gridappsd-docker-build/23)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
...
-  * List open pull requests
-  * Create the release/$VERSION from develop
-  * Test the releases/$VERSION
-  * From the releases/$VERSION branch, create the pull request to master
-  * Assign and close the pull requests
-  * After the pull request's are approved then create the Tagged Releases
+
+To create a release, the create_release.sh script is run multiple times, between each run there are manual testing and validation steps.
+
+1.  Clone the build repository
```
Comment (craig8) [inline README.md]: should make the point that these should be parallel not embedded within

e.g. 
/root/gridappsd-docker
/root/gridappsd-docker-build

not 
/root/gridappsd-docker/gridappsd-docker-build

File: README.md [hunk 2]
```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
...
+
+1.  Clone the gridappsd-docker repository
+```
+git clone https://github.com/GRIDAPPSD/gridapspd-docker
+```
+
+1.  Edit the create_release.sh script to update the version and add GitHub username and token.
+```
+cd gridappsd-docker-build/scripts/release
+vi create_release.sh
```
Comment (craig8) [inline README.md]: I assume there is a VERSION named in here...can we have that in here as what we should change

File: README.md [hunk 3]
```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
...
+
+1.  Verify the containers were built and test the releases_VERSION
+```
+cd ../../../gridappsd-docker
+./run.sh -t releases_VERSION
+```
+
+1.  Run the create_release.sh script to create the pull requests
+```
+cd ../gridapspd-docker-build/scripts/release
```
Comment (craig8) [inline README.md]: so we have create_release.sh in both gridappsd-docker and gridappsd-docker-build?

File: README.md [hunk 4]
```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
...
+./create_release.sh
+```
+ - Update the gridappsd/blazegraph:master container
+ - Create the pull requests from releases/VERSION to master
+
+1.  Assign and merge the pull requests on GitHub    
+
+1.  Run the create_release.sh script to create the tagged releases
+```
+./create_release.sh
```
Comment (craig8) [inline README.md]: hmmm...no modifications, but it will do a different thing with the same command??

Comment (tonya1) [reply README.md]: Correct, the script keeps track of it's state creating the different pieces for the release.

File: README.md (hunk 3 above)
Comment (tonya1) [reply README.md]: No the script is only in gridappsd-docker-build. I included the gridapspd-docker for the references to the run script for testing.

File: README.md (hunk 1 above)
Comment (tonya1) [reply README.md]: I will update to make that more clear.

File: README.md (hunk 2 above)
Comment (tonya1) [reply README.md]: I will update to make that more clear.

**Pull request merged**